### PR TITLE
docs(getting-started): add information about slsa attestor

### PR DESCRIPTION
**Pull request by rrey**
## What this PR does / why we need it

This PR results in a short discussion on [Slack](https://cloud-native.slack.com/archives/C068F87H1MF/p1716556723201989?thread_ts=1714982522.789209&cid=C068F87H1MF).

The Provenance being the very first requirement from SLSA, it seems quite appropriate to mention how Witness helps achieve this easily.

## Acceptance Criteria Met

- [ ] Docs changes if needed
- [ ] Testing changes if needed
- [ ] All workflow checks passing (automatically enforced)
- [ ] All review conversations resolved (automatically enforced)
- [ ] [DCO Sign-off](https://github.com/apps/dco)

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rrey** [comment]
Changing to draft to see if I find other issues while testing witness.
I'll remove the draft flag at the end of the day if I do not find anything else.

File: docs/tutorials/getting-started.md [hunk 1]
```diff
@@ -47,13 +47,16 @@ verify:
...
+- Witness has a set of attestors that are always run. You can see them in the output of the `witness attestors list` command.
 - Defining step names is important, these will be used in the policy.
 - This should happen as a part of a CI step
 
 ```
-witness run --step build -o test-att.json -- go build -o=testapp .
+witness run --step build -o test-att.json -a slsa -- go build -o=testapp .
 ```
 
+><span class="tip-text">💡 Tip: The `-a slsa` option allows to generate the [SLSA Provenace](https://slsa.dev/spec/v1.0/provenance) predicate in the attestation. This is a mandatory requirement for SLSA 1</span>
```
Comment (matglas) [inline docs/tutorials/getting-started.md:58]:
```suggestion
><span class="tip-text">💡 Tip: The `-a slsa` option allows to generate the [SLSA Provenace](https://slsa.dev/spec/v1.0/provenance) predicate in the attestation. This is a mandatory requirement for SLSA Level 1</span>
```

File: docs/tutorials/getting-started.md [hunk 2]
```diff
@@ -47,13 +47,16 @@ verify:
...
+- Witness has a set of attestors that are always run. You can see them in the output of the `witness attestors list` command.
 - Defining step names is important, these will be used in the policy.
 - This should happen as a part of a CI step
 
 ```
-witness run --step build -o test-att.json -- go build -o=testapp .
+witness run --step build -o test-att.json -a slsa -- go build -o=testapp .
 ```
 
+><span class="tip-text">💡 Tip: The `-a slsa` option allows to generate the [SLSA Provenace](https://slsa.dev/spec/v1.0/provenance) predicate in the attestation. This is a mandatory requirement for SLSA Level 1</span>
```
Comment (matglas) [inline docs/tutorials/getting-started.md:58]: I think it might be better to add `--attestor-slsa-export` too. This is the expected Provenance Attestation.

Comment (rrey) [reply docs/tutorials/getting-started.md:58]:
```suggestion
><span class="tip-text">💡 Tip: The `-a slsa` option allows to generate the [SLSA Provenace](https://slsa.dev/spec/v1.0/provenance) predicate in the attestation. The `--attestor-slsa-export` option allows to write the Provenance in a dedicated file. This is a mandatory requirement for SLSA Level 1</span>
```

File: docs/tutorials/getting-started.md [hunk 3]
```diff
@@ -47,13 +47,16 @@ verify:
 ><span class="tip-text">💡 Tip: You can upload the recorded attestations to an [Archivista](https://github.com/in-toto/archivista) server by using the `--enable-archivista` flag!</span>
 - The `-a {attestor}` flag allows you to define which attestors run
 - ex. `-a maven -a gcp -a gitlab` would be used for a maven build running on a GitLab runner on GCP.
+- Witness has a set of attestors that are always run. You can see them in the output of the `witness attestors list` command.
 - Defining step names is important, these will be used in the policy.
 - This should happen as a part of a CI step
 
 ```
-witness run --step build -o test-att.json -- go build -o=testapp .
+witness run --step build -o test-att.json -a slsa -- go build -o=testapp .
```
Comment (rrey) [inline docs/tutorials/getting-started.md:55]:
```suggestion
witness run --step build -o test-att.json -a slsa --attestor-slsa-export -- go build -o=testapp .
```

File: docs/tutorials/getting-started.md (hunk 2 above)
Comment (matglas) [reply docs/tutorials/getting-started.md:58]: I will come back to it, I'm not sure yet. But maybe a maintainer has some input here too

Comment (matglas) [reply docs/tutorials/getting-started.md:58]: Thought about it a bit. I think this is good for now. An additional page dedicated to SLSA inside the concept section could be helpful.

**Review by jkjell — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**